inventory/helpers.go

```diff
@@ -155,6 +168,18 @@ func addMongoDBExporter(t *testing.T, body agents.AddMongoDBExporterBody) *agent
 	return res.Payload
 }
 
+func addPostgresExporter(t *testing.T, body agents.AddPostgresExporterBody) *agents.AddPostgresExporterOKBody {
```

Comment:
Note that this function accepts `body` by value.
Please be consistent. Passing `structs` by value in general case is slightly more preferred, but consistency is much more important.